src/main/java/org/sagebionetworks/web/client/widget/entity/download/Uploader.java

```diff
@@ -952,22 +956,26 @@ public void cancelClicked() {
    * Private Methods
    */
   private void refreshAfterSuccessfulUpload(String entityId) {
-    jsClient.getEntity(
-      entityId,
-      OBJECT_TYPE.FileEntity,
-      new AsyncCallback<Entity>() {
-        @Override
-        public void onSuccess(Entity result) {
-          entity = result;
-          uploadSuccess();
-        }
+    if (entityId != null) {
```

Comment:
If the entityId is set, then do what we used to (get the entity and call success).  If not, just call uploadSuccess.